src/explore-education-statistics-common/src/modules/charts/util/__tests__/createDataSetCategories.test.ts

```diff
@@ -77,8 +78,18 @@ describe('createDataSetCategories', () => {
       publicationName: 'Pupil absence in schools in England',
       subjectName: 'Absence by characteristic',
       timePeriodRange: [
-        { code: 'AY', label: '2014/15', year: 2014 },
-        { code: 'AY', label: '2015/16', year: 2015 },
+        new TimePeriodFilter({
+          code: 'AY',
+          label: '2014/15',
+          order: 0,
+          year: 2014,
+        }),
+        new TimePeriodFilter({
+          code: 'AY',
+          label: '2015/16',
+          order: 1,
+          year: 2015,
+        }),
```

Comment:
This doesn't look right. We're working with a `TableDataResponse`, which should be the pure API response and shouldn't have any class versions of the filters as it won't have been passed into `mapFullTable` yet.

`TableDataReponse` also uses `TimePeriodOption`, which shouldn't have an  `order` field as the order is inferred from the ordering of the `timePeriodRange` array itself.